ui/src/app/domains/views/DomainsList/DomainsTable/DomainsTable.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { mount } from "enzyme";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+
+import DomainsTable from "./DomainsTable";
+
+import {
+  domain as domainFactory,
+  domainState as domainStateFactory,
+  rootState as rootStateFactory,
+} from "testing/factories";
+
+const mockStore = configureStore();
+
+describe("DomainsTable", () => {
+  it("can update the sort order", () => {
```

Comment:
I'm not really sure this test is necessary as it's testing the functionality of an external library component. You could probably have gotten away with checking that the `sortable` key was passed through to the table component. It's fine to leave it now but something to think about in the future.